### ISSUE TITLE
Program executable without admin rights

### ISSUE THREAD

**Issue by Be-Man** [WindowsGSM/WindowsGSM#113]
I would be happy if the program could manage without admin rights, then I would use it too.
I come from IT and you never give programs admin rights if they don't need them 100%. For a game server manager it makes no sense to assign admin rights. If functions require admin rights, such as firewall entries, then these should only be requested when running this case.

One must never forget that attackers like to take advantage of something like this if they are given a free hand! (e.g. with the NotPetya hack and this was done using a tax software from "me Doc")
https://darknetdiaries.com/episode/54/

**Comment by BattlefieldDuck** [comment]
Agree, WindowsGSM requires admin rights because it had a Firewall script that requires admin rights, I guess the next version of WindowsGSM handles it in a better way.

**Comment by Be-Man** [comment]
But also think of offering the possibility of entering this manually into the firewall in order not to need admin rights at all, for those who prefer it that way.

E.g. with a list of which ports have to be opened or which rules would be added.

**Comment by ksduster** [comment]
This is still a huge issue, especially with the hackers going after port 7777 on most gaming servers. 
This really needs to be escalated and resolved to run as a standard user and not as admin giving system level access.
